var/spack/repos/builtin/packages/intel-mkl/package.py

```diff
@@ -29,26 +29,18 @@
 
 
 class IntelMkl(IntelInstaller):
-    """Intel Math Kernel Library.
-
-    Note: You will have to add the download file to a
-    mirror so that Spack can find it. For instructions on how to set up a
-    mirror, see http://spack.readthedocs.io/en/latest/mirrors.html.
-
-    To set the threading layer at run time set MKL_THREADING_LAYER
-    variable to one of the following values: INTEL (default), SEQUENTIAL, PGI.
-    To set interface layer at run time, use set the MKL_INTERFACE_LAYER
-    variable to LP64 (default) or ILP64.
```

Comment:
(we explicitly specify `mkl_integer` and `mkl_threading` libs to be used from MKL)